servers/republik/graphql/resolvers/Comment.js

```diff
@@ -63,11 +63,11 @@ const mdastToHumanString = (node, length = 500, string = '', done = false) => {
 }
 
 module.exports = {
-  discussion: ({ discussionId }, args, { pgdb, discusssion }) => {
-    if (discusssion) {
-      return discusssion
+  discussion: ({ discussion, discussionId }, args, context) => {
+    if (context.discusssion) {
+      return context.discusssion
     }
-    return pgdb.public.discussions.findOne({ id: discussionId })
+    return discussion || context.loaders.Discussion.byId.load(discussionId)
```

Comment:
Generally it should no longer be necessary to cache things in context or on resolving objects. Always load from data loader it caches it.